### ISSUE TITLE
Bad position of sticky toolbar

### ISSUE THREAD

**Issue by jaimestuardo** [froala/wysiwyg-editor#3008]
##### Expected behavior.
Toolbar to be placed correctly.

##### Actual behavior.
Toolbar is placed separated from the editor.

![froala_error](https://user-images.githubusercontent.com/9195659/43960150-1ddc0c9e-9c7f-11e8-82cf-deb3d91c261a.png)

Red border delimits TextArea, including label.

##### Steps to reproduce the problem.
Creating a Bootstrap Dialog and placing the texarea inside it.

This behaviour only happens when control has no focus on first dialog box loading.

This is the actual HTML code (code is not correcly inserted with github editor, I hope you can read it)

`<head>
    <meta name="viewport" content="width=device-width" />
    <title>Agregar Observaciones</title>

    @Styles.Render("~/Content/froala")
    @Scripts.Render("~/bundles/froala")

    <script type="text/javascript">
        $(document).ready(function () {
            $('textarea').froalaEditor({
                language: 'es',
                height: 300,
                zIndex: 9999,
                toolbarButtons: ['fullscreen', 'bold', 'italic', 'underline', '|', 'fontFamily', 'fontSize', 'color', 'inlineStyle', 'paragraphStyle', '|', 'paragraphFormat', 'align', 'formatOL', 'formatUL', 'outdent', 'indent', 'quote', '-', 'insertLink', 'insertImage', 'insertTable', '|', 'insertHR', 'selectAll', 'clearFormatting', '|', 'print', 'spellChecker', 'help', 'html', '|', 'undo', 'redo', 'code_view'],
                codeMirror: window.CodeMirror,
                codeMirrorOptions: {
                    indentWithTabs: true,
                    lineNumbers: true,
                    lineWrapping: true,
                    mode: 'text/html',
                    tabMode: 'indent',
                    tabSize: 4
                },
                imageEditButtons: ['imageDisplay', 'imageAlign', 'imageRemove'],
                imageManagerLoadURL: '@Url.Action("ListImages", "Parametro", new { area = "Backend" })',
                imageUploadURL: '@Url.Action("UploadImage", "Parametro", new { area = "Backend" })',
                imageManagerDeleteURL: '@Url.Action("DeleteImage", "Parametro", new { area = "Backend" })'
            })
        });
    </script>
</head>
<body>
    <div class="x_panel">
        <div class="x_content">
            <div id='observacion_ajax_loader' style="position: fixed; left: 50%; top: 50%; display: none; z-index: 9999">
                <img src="~/Images/ajax-loader.gif" />
            </div>
            <div style="border: solid 1px red">
                <form id="frmObservacion" action="'@Url.Action("AddNotesToReport", "Servicio", new { Area = "Gestion" })';" class="form-horizontal form-label-left form-popup" novalidate="novalidate">
                    <div class="item col-md-12 col-sm-12 col-xs-12 form-group">
                        <label for="observaciones">Ingrese Observaciones (opcional)</label>
                        <textarea class="form-control" id="observaciones" name="observaciones" placeholder="Observaciones" cols="20" rows="6"></textarea>
                    </div>
                </form>
            </div>
        </div>
    </div>
</body>
</html>`

By using developer tools, I could realize that element has the top:0 style.. the same as .fr-toolbar.fr_top.

If I remove "top:0" style, it works.

![froala_error_2](https://user-images.githubusercontent.com/9195659/43960396-e0e98e78-9c7f-11e8-8983-e1ce81e4e22b.png)

##### OS.
Any

##### Browser.
Any

**Comment by stefanneculai** [comment]
Do you think it would be possible to make a jsFiddle to highlight the problem? That would speed-up things a lot. Here is a basic jsFiddle you could use as a starting point: https://jsfiddle.net/froala/wc5c3jhk/.

**Comment by jaimestuardo** [comment]
It is not possible to do that.... I tried but without success. Besides froala plugin I needed bootstrap dialog.... I  didn't really know how to add all of them to jsFiddle. 

However, I know where the problem is (maybe). It seems when I used toolbarSticky: false, problem was solved. I have realized that there is a class applied to toolbar control named sticky-on was causing the problem.

When that class appears, the toolbar was placed to the top of dialog box, instead of the top of the containing DIV.

Is somewhere the uncompressed version of the froala javascript files? By having only the minimized version, it is impossible to fix problems by myself. In this case, fortunately it could be solved by that setting. I will continue testing it.

Regards,
Jaime
